### ISSUE TITLE
Can't edit or create new images using Chrome on Windows 10

### ISSUE THREAD

**Issue by mooreds** [renanlecaro/mermaid-gdocs#31]
I have an existing document with some diagrams (all sequence diagrams). They render fine.

However, when I try to create a new diagram or update an existing one, there's an issue.  When updating, I'm able to make a change to the diagram via `Edit selected chart` and have it reflected, but when I click `Update`, I see this error message:

`ScriptError: We're sorry, a server error occurred. Please wait a bit and try again.`

I see a similar error message when creating a new diagram and clicking `Insert` with the default graph diagram code. I saw this error message yesterday and this morning.

I've tried:

* uninstalling and reinstalling the extension
* adding a chart in a new document

When looking at the error console, I don't see anything obvious, but there is a warning message when I first start up the extension: `There was an error during the transport or processing of this request. Error code = 404, Path = /wardeninit` 

When clicking save, I see:

```
graph LR
  A -->B data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOQAAABiCAYAAABAvn9IAAAAAXNSR0IArs4c6QAACaZJREFUeF7tnXtsFEUcx7/HlZ6gvKMRbGhKxEAggKUgMfjioRKUACIBiVGrf1AoCC3xUfvioVUElEdKSCwiRlQiEvxD4zMkNgSVIBpDFNEKNCCiaBRoWq6cmbXX7J13dm/md3vb9bv/EGB+M7/5zH5ud2dm7wLgQQIk4BkCAc9kwkRIgARAIXkSkICHCFBIDw0GUyEBCslzgAQ8RIBCemgwmAoJUEieAyTgIQIU0kODwVRIgELyHCABDxGgkB4aDKZCAhSS5wAJeIgAhfTQYDAVEqCQPAdIwEMEKKSHBoOpkACF5DlAAh4ioC1kbdGxPq2hlv0e6ot+KgF8tOiFwQv1K2CkBIGNS74/DCAoUVcm6wg2Z49bsDn3d50cTIU8q9Oo52ICqKWQmR+VNiGHZj4TswyCzdl9MyZkqFsQkwtzzHqQoejTDU344t1fAAqZoRGIbTYq5NSiXAS6eCKllJL4cGsjmptakXEhZ5bmpZS4VwofPfgnhfTKYACwC9mzX1cPZeYslbfXNlBIZ6gSl6KQJvTkYymkulnTPNomdc6qW1ZeITUhMiyGAIWkkLxl9dCHAoWkkBSSQooR4DOkIUo+QxoCFA7nFZJXSF4hhaUyqY5CUkgKaWKQcCyFpJAUUlgqk+ooJIWkkCYGCcdSSApJIYWlMqmOQlJICmlikHAshaSQFFJYKpPqKCSFpJAmBgnHUkgKSSGFpTKpjkJSSAppYpBwLIWkkBRSWCqT6igkhaSQJgYJx1JICkkhhaUyqY5CUkgKaWKQcCyF9LiQv/12BqtXV2L+/BLk5Q0WHn6Ar1+JIzWqUErInTu3o77+Y1y4cB4nTvwUk1N2djYGDboOY8bciIkTp+LqqwcgEND+4oyYun3/PuTnn9ejomIJHnxwAebOLTQa7ETBFFIcqVGFUkLW1W3Ezp2vdJhLVlYWZs26H/fd9whCoVCH5Tsq4GshL15swcaNz+L999/BqFFjUF7+HHr06NkRk5T+n0KmhCvthdMhpLoC9urVpz33RFfNwsJizJ79gPGV0tdCNjR8j6eeWgx126o+yWpqajFiRL7oSUEhRXEaV5YOIUtKKnHHHdNicjt//hx27KjDW2+9av17Ts5ArFq1Af37m32/sG+FjEQieOONl7FtW207yHvumYeHH16MYFDum+YppLFDohW4JaRKWkm5Zk019u3ba/Vh5coXMXbseKP++FZIdVWsri7FkSPqpx7+OQYOzMPKleuth3Cpg0JKkUxcT0FBQXUkEqkCcCEQCDwfDodfPHTo0B/JWnVTSJWD/Vkz0ZU0VTq+FXLv3g9QU1OGbt264667ZmH37h0Ih8N48slncOutt6fKKWl5CimGMmFFbUKWAOgBoAVA+L/EdFPI1tZWbNmyDnv2vCn2SORLIZuaLli3EvX1nyA//wbMn19q/V1dLW+77U4sXVqOUOgykTOJQopgTFpJnJDRcknFdFPIY8d+xIoVy9DYeByjR4+zPuxNJw19KaR9MkfJOG3a7PZPsn79rsTTT28QW5OkkBkRMkbMSCSy+tKlS+vVrawbQp479xe++uoAXnppA06ePGFN6FRWrkFu7iBjGL4T0j6ZY5cvuh6piEmuSUaF/PbUB9h/dGsqA3IplcL/47Jqxd3xqvuM0WvRq/s1UL9+ZfJjO07WIdUGgQkTpmDevEdw1VX9RYbId0LaJ3PGj5+AZcuqrefIX3/9BZWVS/DDD0dE1yQNhBQZQFYSS8BtIdWs6owZczFs2EgEBH7/zndCRidz1DDZZ73S8QCu2qCQ3vpISIeQ8RsDWlqare10apIwetx88yQsXPgYevfuawTEV0LaJ3MSLdR++unHWLXqcQuY1JqkgZC8ZXV26mb8ljXRcob6gP/mmy+xffsW6091TJo0FcXFj1t3ZLqHr4Q8fPhrlJUVQ4mpdlYsWvQEunbNbmfz888nUVHxKI4fbxBbk+Skju6p5ywuySxrNLgZQKt9GcSNSR175mfOnEZ1dQmOHv3OWvpYvnwdCgpudNa5BKV8I6SazFG7ctTuHHWofas33TQxpsvh8EVs2rQa77232/p3iTVJCql97jkKTCLkv0SMVua2kKrd11/f2r4jbM6ch6xJQ923P3wjpH0y57925NhvWyXWJCmkI6+0C8UJmVTETAqpXl5Yt26FlYLpOeUbIe2TOU5HX2JNkkI6pa1Xzutb51SvKGTc2Nonc1IddtM1SQqZKvHUyufn5y/p0qVL7472sGbqChn/EsOUKTNQXPwYsrK6ptbRttK+uELaJ3PUYq26ZQ0Gs5ICsb/Ppt6TrKpag+7dL9cCSCG1sKUtyO1nSPv2OdUpNct69933avev0wuppp/r6jZg167XLAiJZlfj6Zw61Yjy8sXWHkQ1M7Z+/TZce+0QLYgUUgtb2oLcElJNEB48+Bk2b15rbZ9Th9o6p2ZZTd6J7PRCxsvlZNo5fra1qKgU06fP1TpJKKQWtrQFpUPIK67oiQEDctpnThN9Y4AqU17+LK6/fqxR3zq9kPYH6qFDR2D58rUxX7eQjM6BA/tQVVVi7ba45ZbJKCur0QJJIbWwpS0oHUJ2lKy6Mi5dWoEhQ4ZrL3dE2+jUQqoHarXzRr1mpQ41QaPWgZysAdmXSRRIdduqc1BIHWrpi3FLSLWdbuTIAmuZY/jwUTEbUEx616mFNOm4VCyFlCIpU4+UkDLZpF4LhUydWUwEhTQEKBxOIVN4Vy2efW3RsT6toZazoW5BzCzNEx4ad6qjkO5wdtoKhaSQ/CkBp7a4UI5CUkgK6YJoTpugkBSSQjq1xYVyFJJCUkgXRHPaBIWkkBTSqS0ulKOQFJJCuiCa0yYoJIWkkE5tcaEchaSQFNIF0Zw2QSEpJIV0aosL5SgkhaSQLojmtAkKSSEppFNbXChHISkkhXRBNKdNUEgKSSGd2uJCOQpJISmkC6I5bYJCUkgK6dQWF8pRSApJIV0QzWkTFFJIyMmFOU6Ze6rc6YYmCumhEbELKfBzja737MOtjWhuakWwObvvgs25v+sk4PjXbeMrj35jgE6jnosJoHbRC4MXei6v/1lCUSE7e7czKeT+zg7Pyj+Ajyhk5keyTchg5jMxyyDYnD3O9SukWcqMJgESSERA+5aVOEmABOQJUEh5pqyRBLQJUEhtdAwkAXkCFFKeKWskAW0CFFIbHQNJQJ4AhZRnyhpJQJsAhdRGx0ASkCdAIeWZskYS0CZAIbXRMZAE5AlQSHmmrJEEtAlQSG10DCQBeQIUUp4payQBbQIUUhsdA0lAngCFlGfKGklAm8Df9ItC6pIorMkAAAAASUVORK5CYII=
2382548268-warden_bin_i18n_warden.js:107 Net state changed from IDLE to BUSY
2382548268-warden_bin_i18n_warden.js:107 Net state changed from BUSY to IDLE
```

but no other errors.

I'm using `Version 112.0.5615.138 (Official Build) (64-bit)` on Windows 10. I tested on `Version 112.0.5615.137 (Official Build) (x86_64)` on macos (intel) and it works, weirdly enough.

**Comment by mooreds** [comment]
Hmmm. I tried it in another browser (MS Edge, `Version 112.0.1722.64 (Official build) (64-bit)`) and it failed with

`ScriptError: Authorization is required to perform that action.`

But then I removed all my google accounts on Edge (had 2) and then signed in again with only one account and it worked. Chrome had multiple accounts too (both today when it failed and a few weeks ago when the diagramming plugin worked). The Macos chrome browser had only one account set up when I tested it.

So perhaps the issue is when a user has multiple Google accounts?
